packages/ui-extensions/src/surfaces/customer-account/components/Menu/Menu.ts

```diff
@@ -6,6 +6,14 @@ export interface MenuProps extends IdProps {
    * A label to describe the purpose of the menu that is announced by screen readers.
    */
   accessibilityLabel?: string;
+  /**
+   * Callback to run when the Menu is opened
+   */
+  onOpen?: () => void;
+  /**
+   * Callback to run when the Menu is closed
+   */
+  onClose?: () => void;
```

Comment:
I first wanted to extend `Pick<PopoverProps, 'onOpen' | 'onClose'>` but this would say "Popover" instead of "Menu".